### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by stutisk**
Added Wishlist Page

### CODE REVIEW

File: src/Components/Cards/Cartpagecard.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 import "../../styles/Cartpage.css";
 import "../../styles/ProductListing.css";
 import React from "react";
 import { useCart } from "../../Context/CartContext";
 import { HiMinusCircle, HiPlusCircle, FaBookmark } from "../Icons";
-
+import {Link}  from "react-router-dom";
 const Cartpagecard = ({ product }) => {
   const { state, dispatch } = useCart();
   console.log(state.wishlist);
```
Comment (shuklahimanshu81) [inline src/Components/Cards/Cartpagecard.jsx:10]: Remove console statements

File: src/Components/Cards/Wishlistcard.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import "../../styles/Cartpage.css";
+import "../../styles/ProductListing.css";
+import React from "react";
+import { useCart } from "../../Context/CartContext";
+
+const Wishlistcard = ({ product }) => {
+  const { dispatch } = useCart();
+  //   console.log(state.wishlist);
+  //   console.log(state.cart);
+  return (
```
Comment (shuklahimanshu81) [inline src/Components/Cards/Wishlistcard.jsx:11]: Remvoe comments.

File: src/Components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -13,13 +13,13 @@ const Navbar = () => {
...
+            <a className=" nav-link padding " >
               Women
             </a>
-            <a className="nav-link padding" href="">
+            <a className="nav-link padding" >
               Men
             </a>
-            <a className="nav-link padding " href="">
+            <a className="nav-link padding ">
               Kids
```
Comment (shuklahimanshu81) [inline src/Components/Navbar/Navbar.jsx:23]: Better to use Link

**Review by shuklahimanshu81 — APPROVED** [review]
Everything looks good to me. Good to merge.

File: src/Components/Cards/Wishlistcard.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          <div className="delivery details">
+            <div className="delivery-details m-5">{product.brand}</div>
+          </div>
+          <div className="price-container m-5">
+            <div className="price">{product.price}</div>
+          </div>
+          <div className="delivery details">
+            <div className="delivery-details m-5">
+              {product.deliverydetails}
+            </div>
```
Comment (talrejah02) [inline src/Components/Cards/Wishlistcard.jsx:32]: You can destructure "product" to avoid using dot operators for clean code.

**Review by talrejah02 — APPROVED** [review]
easily understandable, good for merge.

**Pull request merged**